src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/SecurityTokenRequest.cs

```diff
@@ -41,6 +41,11 @@ class RequestSecurityTokenResponse
         /// Gets or sets the access token.
         /// </summary>
         public string Token { get; set; }
+
+        /// <summmary>
+        /// Gets or sets the token expiration
+        /// </summary>
+        public int ExpiresOn { get; set; }
```

Comment:
Why is this being added anyways? You aren't changing anything with the security token request messages that I can see.